app/questionnaire/rules.py

```diff
@@ -62,6 +62,15 @@ def evaluate_condition(condition, answer_value, match_value):
     """
     answer_and_match = answer_value is not None and match_value is not None
 
+    if condition in {"equals", "not equals", "equals any", "not equals any"}:
+
+        answer_value = casefold(answer_value)
+
+        if isinstance(match_value, list):
+            match_value = list(map(casefold, match_value))
```

Comment:
match_value = list(map(casefold, match_value)) if list(map(casefold, match_value)) else casefold(match_value) 

maybe?

Reply:
I think this would fail for not iterable data